libcxx/test/std/containers/sequences/vector/vector.cons/copy.move_only.verify.cpp

```diff
@@ -16,5 +16,5 @@
 
 void f() {
     std::vector<MoveOnly> v;
-    std::vector<MoveOnly> copy = v; // expected-error-re@* {{{{(no matching function for call to 'construct_at')|(call to implicitly-deleted copy constructor of 'MoveOnly')|(call to deleted constructor of 'MoveOnly')}}}}
+    std::vector<MoveOnly> copy = v; // expected-error-re@* {{{{(no matching function for call to '__construct_at')|(call to implicitly-deleted copy constructor of 'MoveOnly')}}}}
```

Comment:
It's unfortunate that `__construct_at` appears here, but I think that suggests we're testing too much of the compiler message. 

A test should only fail when the behavior under test no longer holds. Is the name part of the behavior under test?